addons/event_sale/report/event_event_templates.xml

```diff
@@ -14,7 +14,7 @@
             </div>
             <div t-if="attendee and attendee.sale_order_line_id.price_total">
                 <div class="o_event_full_page_ticket_font_faded o_event_full_page_ticket_small_caps fw-bold">Price</div>
-                <div class="o_event_full_page_ticket_small" t-field="attendee.sale_order_line_id.price_total"
+                <div class="o_event_full_page_ticket_small" t-field="attendee.sale_order_line_id.price_subtotal"
```

Comment:
Tell me if I'm wrong but if I understood correctly, here we want to display the price of the ticket (taxes excluded).

If so then this only works when there is 1 ticket per sale order line. But with multiple tickets `price_subtotal` will not display the price of the ticket but the total for the sale order line without the taxes.

If we want to display the price of the ticket we print then we should use `attendee.sale_order_line_id.event_ticket_id.price`

Reply:
Hello @jeh-odoo 

Changes done.

if we consider the previous version, it will break the stable policy.
what do you think?

Thanks

Reply:
Thanks for the changes !
Well the only point I'm not sure is if it's really inevitable :thinking: 
I guess not, it does not break anything, only displays a "false" info.
You can keep it like that then :smile: 

Cheers!

Reply:
Hello @jeh-odoo 

In V14.0 we don't have such a kind of report.
it is started from V15.0

should I re-target it to 15.0?

Thanks

Reply:
Hello @nla-odoo 
If we really want to fix the price displayed on the ticket for stable versions than yes we could re-target to 15.0.
But I'm not sure if it's really necessary, even if it should not break anything to fix this in stable.

I guess you can retarget it to 15.0 for now and we will see if we keep it that way before the merge.

Cheers!